proto/bff/models/v1/models.proto

```diff
@@ -52,6 +52,8 @@ message Collaborator {
     // Metadata as RFC3339Nano Timestamps
     string created_at = 10;
     string modified_at = 11;
+    string joined_at = 12;
+    string last_login = 13;
```

Comment:
The last login timestamp will be returned on `ListOrganizations` requests so that the frontend can order the organizations for the user by most recently used.

Reply:
So the frontend will order the collaborators multiple organizations by the last login of the collaborator?

Reply:
The problem is that is has to be per-user and the `Organization` record has multiple collaborators on it (in the map). So what I was thinking was that on `ListOrganizations` as we are iterating through the organizations we would just lookup the collaborator from the map by ID (which is a hashed version of the user's email address) and add the timestamp to the reply for that organization.